pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -407,15 +397,6 @@ func NewNodeLifecycleController(
 	nc.podLister = podInformer.Lister()
 	nc.nodeLister = nodeInformer.Lister()
 
-	if !utilfeature.DefaultFeatureGate.Enabled(features.SeparateTaintEvictionController) {
```

Comment:
should this code exist for a while to allow for emulated versions? Or there is no emulated versions on a controller? I am not sure

Reply:
reverted. I forgot the requirements of `compatibility version`